src/checklists/backend.md

```diff
@@ -23,15 +23,15 @@
 | pablo             | 🍏     | 🍏               | 🍏           |
 | payot             | 🍏     | 🍏               | 🍏           |
 | pechkin           | 🍅     | 🍏               | ❓           |
-| roberto           | ❓     | ❓               | ❓           |
+| roberto           | 🍏     | 🍏               | 🍅           |
 | scrooge           | 🍅     | 🍏               | ❓           |
 | shiva             | 🍏     | 🍏               | 🍏           |
 | skyler            | 🍏     | 🍏               | 🍏           |
 | tigerdocs         | 🍅     | 🍅               | ❓           |
 
 ### Toolkit
 
-| Service / Feature | wrappers | logger | ACL | lambda | eventBus | migrate | retry DLQ | sentry | telemetry | ssmWrapper | sns-via-sqs | api3 |
+| Service / Feature | wrappers | logger | ACL | lambda | eventBus | migrate | retry DLQ | sentry | telemetry | ssmWrapper | sns via sqs | api3 |
```

Comment:
otherwise looks ugly

<img width="212" alt="Screenshot 2023-09-05 at 10 19 35" src="https://github.com/OsomePteLtd/principles/assets/2411588/e461b00f-ed9b-46d3-959c-a046f4ad7549">